arithmetization/src/main/java/net/consensys/linea/zktracer/module/romlex/RomLex.java

```diff
@@ -50,14 +52,17 @@
 import org.hyperledger.besu.evm.worldstate.WorldView;
 
 @Accessors(fluent = true)
+@RequiredArgsConstructor
 public class RomLex
-    implements Module, PostOpcodeDefer, ImmediateContextEntryDefer, ContextExitDefer {
-  private static final RomChunkComparator ROM_CHUNK_COMPARATOR = new RomChunkComparator();
+    implements StatelessModule<RomChunk>,
+        PostOpcodeDefer,
+        ImmediateContextEntryDefer,
+        ContextExitDefer {
 
   private final Hub hub;
 
   @Getter private final StackedSet<RomChunk> chunks = new StackedSet<>();
```

Comment:
Why the name is different here from other modules ? sometimes I see `chunks`, sometimes `operations`.
If the name of the attribute is `operations`, the getter should be : `getOperations`. If the name is `chunks`, the getter should be `getChunks()`

Reply:
forgotten renaming, will switch to operations as all others